ml-agents/mlagents/trainers/learn.py

```diff
@@ -75,6 +80,24 @@ def run_training(
             docker_target_name=docker_target_name
         )
 
+    sampler_config = None
+    lesson_length = None
+    if sampler_file_path is not None:
+        sampler_config = load_config(sampler_file_path)
+        if ("episode-length") in sampler_config:
+            lesson_length = sampler_config.pop("episode-length")
+            if lesson_length <= 0:
```

Comment:
Is there a chance that lesson_length is not a number ? If testing for non positive numbers, we should also test those.